README.md

```diff
@@ -188,10 +188,16 @@ The Python client enables you to create [Scoped Keys](https://keen.io/docs/secur
 
 ### Changelog
 
+##### 0.3.15
++ Added better error handling to surface all errors from HTTP API calls 
+
 ##### 0.3.14
 + Added compatibility for pip 1.0
 
 ##### 0.3.13
++ Added better error handling to surface all errors from HTTP API calls 
```

Comment:
0.3.13 is repeated here, probably you meant to take out these lines and just list it as 0.3.15?